OpenRA.Mods.Common/Scripting/Properties/TransportProperties.cs

```diff
@@ -42,9 +42,9 @@ public TransportProperties(ScriptContext context, Actor self)
 
 		[ScriptActorPropertyActivity]
 		[Desc("Command transport to unload passengers.")]
-		public void UnloadPassengers()
+		public void UnloadPassengers(Target? destination = null)
```

Comment:
This one needs to stay nullable because of the way Lua and our custom bridge works.

Reply:
Fixed.